File: BankApplicationAPI/BankApplicationAPI/Models/ComplaintFeedback.cs
namespace BankApplicationAPI.Models;

public partial class ComplaintFeedback
{
    public int FeedbackId { get; set; }

    public int? ComplaintId { get; set; }

    public string? CustomerId { get; set; }

    public DateTime? FeedbackDate { get; set; }

    public byte? FeedbackRating { get; set; }

    public string? FeedbackComments { get; set; }

    public virtual Complaint? Complaint { get; set; }

    public virtual Customer? Customer { get; set; }
}

File: BankApp/src/app/Modules/ComplaintFeedback.ts
import { Complaint } from "./Complaint";
import { Customer } from "./Customer";

export interface ComplaintFeedback {
    feedbackId?: number | null;
    complaintId?: number | null;
    customerId?: string | null;
    feedbackDate?: Date | null;
    feedbackRating?: number | null;
    feedbackComments?: string | null;
    complaint?: Complaint | null;
    customer?: Customer | null;
}